docs/client/operations.md

```diff
@@ -0,0 +1,228 @@
+# <img align="center" src="../images/logo.png">  Calling Operations with Your Java Client
+
+AutoRest provides both synchronous and asynchronous method overloads for each service operation.
+Depending on your swagger definition, operations can be accessed through operation groups (TODO: link to swagger docs) on the client,
+or directly on the client.
+
+## Operation Group vs No Operation Group
+
+If your swagger defines an operation group for your operation (for example, in [this][operation_group_example] swagger, the operation `list`
+is part of operation group `application`), you would access the operation through the operation group getter on the client, `getApplications()`.
+This makes the call `client.getApplications().list()`.
+
+If there's no operation group, as in [this][mixin_example] case, you would access the operation directly from the client
+itself, i.e. `client.getDog()`.
+
+## Regular Operations
+
+### Sync Operations
+
+We will be using the [example swagger][pets_swagger] in our main docs repo. After [initializing][initializing] our client, we
+call our operation like this:
+
+```java
+package com.azure.pets;
+
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.pets.models.Dog;
+
+public static void main(String args[])
+{
+    PetsClient client = new PetsClientBuilder()
+        .credential(new DefaultAzureCredentialBuilder().build())
+        .buildClient();
+
+    Dog dog = client.getDog();
+}
+```
+
+### Async Operations
+
+When calling our async operations, we use our async client, which is suffixed with `AsyncClient` instead of `Client`. Our async operations
+return an [`Observable`][observable], so they need to be subscribed to. Following the [example above](#sync-operations Sync Operations),
+our call to `getDog` looks like this:
+
+```java
+package com.azure.pets;
+
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.pets.models.Dog;
+
+public static void main(String args[])
+{
+    PetsAsyncClient client = new PetsClientBuilder()
+        .credential(new DefaultAzureCredentialBuilder().build())
+        .buildAsyncClient();
+
+    client.getDog().subscribe(
+        d -> dog = d
```

Comment:
Where is `dog` defined? I guess we could do a simple `System.out.println()` on one of the `d`'s properties.